library/registry

```diff
@@ -2,9 +2,21 @@ Maintainers: David Wu (@davidswu),
              Ryan Abrams (@caervs)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
-Tags: 2.6.2, 2.6, 2, latest
-GitFetch: refs/heads/master
-GitCommit: 15dbd1a011dbda8da055d6cea8bc1e7705c32ead
+Tags: 2.7.0, 2.7, 2, latest
+Architectures: amd64, arm64v8, arm32v6
+GitFetch: refs/heads/registry-v2.7.0
+GitCommit: c8fb369af7ecefd44047f5140a5180ac8d643100
+amd64-Directory: amd64
+arm32v6-Directory: arm
+arm64v8-Directory: arm64
+
+Tags: 2.6.2, 2.6
+Architectures: amd64, arm64v8, arm32v6
+GitFetch: refs/heads/registry-2.6
```

Comment:
I don't see a `registry-2.6` branch in docker/distribution-library-image but I do see a `release/2.6` branch that was updated with https://github.com/docker/distribution-library-image/pull/83.

And then the GitCommit will need to be updated (probably `2798058c4eadcbb2510df72d50f01e59f393480b`, but the merge commit is also fine https://github.com/docker/distribution-library-image/commit/fc40f1f1051bb4a42ee4661ccaa190c1bd6c6be9).